Unity-Technologies/com.unity.multiplayer.samples.bitesize
# Multiplayer Use Cases: package/project upgrades

### Description
- Upgraded project to Unity 6 LTS (6000.0.23f1), and packages to the latest versions
- Minor tutorial fix

### Issue Number(s)
- N/A

### How to test/validate

1. Clone this branch
2. Open it with Unity 6 LTS (6000.0.23f1)
3. Go through all tutorials and ensure you can follow them all without issues

### Contribution checklist
